implementations/grafbase/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "fullName": "Grafbase",
+  "language": "Rust/Typescrpt",
```

Since this is not a "traditional" RS / TS server I think it should be under "Other" section

```suggestion
  "language": "Other Solutions",
```